### PR TITLE
[FEAT] Always display navbar

### LINKED ISSUE #57: [FEAT] examples - always display the banner/navbar
New design introduced with #53 
I would be nice to always display the banner to have access to repository and docs links

### PR DESCRIPTION
**Pull request by tbouffard**
closes #57

### CODE REVIEW

File: examples/custom-colors/index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
         <section class="navbar-section">
             <a href="https://process-analytics.github.io/bpmn-visualization-js/" class="mr-2 btn btn-secondary">Docs</a>
             <div class="dropdown mr-2">
-                <a href="#" class="btn btn-primary dropdown-toggle" tabindex="0">
+                <a href="#" class="btn btn-primary dropdown-toggle" tabindex="0" onclick="return false;">
```
Comment (tbouffard) [inline examples/custom-colors/index.html:33]: @aibcmars if you know a better way please commit directly in this PR

Comment (aibcmars) [reply examples/custom-colors/index.html]: yes, I have replaced anchor with span - anchor with href="#" is always going to the top
I have also changed the dropdown display direction so it is nicer(not cut)

File: examples/index.html [hunk 2]
```diff
@@ -208,9 +208,9 @@ <h2>Extensibility</h2>
     </div>
 </section>
 
-<footer class="navbar bg-dark p-2">
+<!--<footer class="navbar bg-dark p-2">-->
```
Comment (aibcmars) [inline examples/index.html:211]: @tbouffard do we need that?

Comment (tbouffard) [reply examples/index.html:211]: No, there is no reason to keep the comment.
I am removing it

Comment (tbouffard) [reply examples/index.html]: done with 9ce9557

**Review by aibcmars — APPROVED** [review]
LGTM

**Pull request merged**